packages/trip-details/src/trip-detail.tsx

```diff
@@ -13,17 +13,8 @@ type Props = {
   summary: ReactElement | string;
 };
 
-// TODO: Remove these two helper methods by moving to semantically correct HTML
-/**
- * Copied from https://stackoverflow.com/questions/50940640/how-to-determine-if-jest-is-running-the-code-or-not
- */
-function isRunningJest() {
-  return process.env.JEST_WORKER_ID !== undefined;
-}
```

Comment:
this hasn't done anything for a long time since the snapshot tests don't run in jest since I did storybook 7 update